server/models.py

```diff
@@ -1503,6 +1503,8 @@ def default_redirect_uri(self):
         return self.redirect_uris[0]
 
     def validate_redirect_uri(self, redirect_uri):
+        if redirect_uri == OAUTH_OUT_OF_BAND_URI:
```

Comment:
I get why we want loopback addresses to automatically be allowed - but why not make users add the URI manually if they want it? 

(I'm not really opposed to it - just want to see if there's some special reasoning for it)